src/ui/public/embeddable/context_menu_actions/build_eui_context_menu_panels.ts

```diff
@@ -57,7 +58,7 @@ function buildEuiContextMenuPanelItemsAndChildPanels({
   const childPanels: EuiContextMenuPanelDescriptor[] = [];
   const actionsForPanel = getActionsForPanel(contextMenuPanelId, actions);
   actionsForPanel.forEach(action => {
-    const isVisible = action.isVisible({ embeddable, containerState });
+    const isVisible = action.isVisible({ embeddable, containerState, uiCapabilities });
```

Comment:
If the embeddable itself knows if it can be edited, I don't think we'll need to pass it the uiCapabilities.